### ISSUE TITLE
Continous scrolling: chapter missing

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#136]
I can't yet reproduce this, but this happened once to me, using develop at 8a73b9622da1e16a0d24de1fe736fbea7689791f. Got screenshot. I'll post more information when I can figure out how to reproduce.

![screenshot_20180526-134332](https://user-images.githubusercontent.com/5811789/40575307-57ae59e2-60eb-11e8-87c4-7f13b928e7f2.png)

**Comment by mjdenham** [comment]
I can quite believe this.  It is one of the reasons I delayed release so long.  Up-scroll is very hard and there are all sorts of potential issues.
But if you can create a repro I will take a look.

**Comment by timbze** [comment]
Device info:
And Bible version: 3.3.389#56bab7d (built 20/08/20 11:35:18)
Android version: 10
Android SDK version: 29
Manufacturer: Google
Model: Pixel 3

Storage Mb free: 32039

Used heap memory in Mb: 8
Max heap memory in Mb: 192

**Had another issue with missing chapter (log attached at bottom). Here's what I did:**
1. In evening opened Eph 4:11 in pinned window 1, and Gill commentary in unpinned window 2
2. Pressed home > get out of app
3. In morning, 6+ hours later, opened And Bible and it shows the following. This is after having used other apps in between and possibly Android memory management has done something to And Bible
![Screenshot_20200903-053207](https://user-images.githubusercontent.com/35117769/92126094-fe0b2800-edbc-11ea-8628-dbedf5efd1df.png)

**Then, scroll up and chapter 3 get's loaded, like this:**
![Screenshot_20200903-053226](https://user-images.githubusercontent.com/35117769/92126149-0cf1da80-edbd-11ea-9cbd-34e8fb51fada.png)

**Line 57 in log:**
`SwordContentFacade: KEY:Eph.4 not found in doc:engKJV2006eb(KJV)`

Note, the crash in the log is issue #818 and not related to this issue.
[logcat (5).txt](https://github.com/AndBible/and-bible/files/5169150/logcat.5.txt)

**Comment by timbze** [comment]
I did not even notice, but log had weird lines about every other verse missing, and now I see that is actually what Bible showed, v2, 4, 6 etc in ch. 5 are missing also.

**Comment by tuomas2** [comment]
Should be reproduced in this logcat: https://support.andbible.org/scp/tickets.php?id=148

**Comment by timbze** [comment]
> I did not even notice, but log had weird lines about every other verse missing, and now I see that is actually what Bible showed, v2, 4, 6 etc in ch. 5 are missing also.

Also reproduced here: https://support.andbible.org/scp/tickets.php?id=201

**Comment by tuomas2** [comment]
Unless the issue is in JSword side, I believe this issue will disappear after #888.

**Comment by tuomas2** [comment]
If I remember correctly someone has seen this still in beta.

**Comment by tuomas2** [comment]
#1057 proves that it's still alive
